src/components/head.js

```diff
@@ -23,10 +24,14 @@ export default (props) => {
     .filter(Boolean)
     .join(" | ")
   const description =
-    (frontmatter && frontmatter.description) || props.description
+    (frontmatter && frontmatter.description) || props.description || "Swizec helps you become a better frontend engineer with books, articles, talks, and workshops";
```

Comment:
Can we update code like this to conditional chaining? I think it works with the pipeline we’ve got

```javascript
frontmatter?.description || props.description || “Default description”
```